### PR TITLE
Umbraco Commerce Docs Updates

### PR DESCRIPTION
**Pull request by mattbrailsford**
## Description

Fixing some broken links, and added a missing docs page for Realtime Shipping Rate setup

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [x] Other

## Product & version (if relevant)

v13

## Deadline (if relevant)

ASAP

### CODE REVIEW

**Comment by alina-tincas** [comment]
Hi @mattbrailsford thank you for the PR 🙌 

We will review it as soon as we can 🙏

File: 13/umbraco-commerce/reference/shipping/realtime-rate-shipping.md [hunk 1]
```diff
@@ -1,2 +1,31 @@
+---
+description: Realtime Rate Shipping in Umbraco Commerce.
+---
+
 # Realtime Rate Shipping
 
+Realtime rate shipping in Umbraco Commerce allows you to define realtime, up to the minute shipping estimates directly from shipping operators. 
+
+## Configuration
```
Comment (eshanrnh) [inline 13/umbraco-commerce/reference/shipping/realtime-rate-shipping.md:9]:
```suggestion
## Configuring Realtime Rate Shipping

To configure Realtime Rate Shipping, follow these steps:
```

File: 13/umbraco-commerce/reference/shipping/realtime-rate-shipping.md [hunk 2]
```diff
@@ -1,2 +1,31 @@
...
+* Click `Create Shipping Method`
+* Choose the shipping provider for thie shipping operator you wish to use
+
+![Choose Shipping Provider](../../media/create_shipping_method.png)
+
+* Chose the `Realtime` calculation mode option
+
+![Choose Shipping Calculation Mode](../../media/create_shipping_method2.png)
+
+* Populate the shipping method name, alias, sku and optional image and tax rate
```
Comment (eshanrnh) [inline 13/umbraco-commerce/reference/shipping/realtime-rate-shipping.md:24]:
```suggestion
8. Enter the **Shipping Method Name**, **Alias**, **SKU**.
9. Select the tax class from the **Tax Class** dropdown list.
10. _[Optional]_ Upload an image.
```

File: 13/umbraco-commerce/reference/shipping/realtime-rate-shipping.md [hunk 3]
```diff
@@ -1,2 +1,31 @@
...
+* Enter any shipping provider API credentials required to connect to the shipping operators API
```
Comment (eshanrnh) [inline 13/umbraco-commerce/reference/shipping/realtime-rate-shipping.md:25]:
```suggestion
11. Enter the shipping provider's API credentials required to connect to the shipping operator's API.
```

File: 13/umbraco-commerce/reference/shipping/realtime-rate-shipping.md [hunk 4]
```diff
@@ -1,2 +1,31 @@
...
+
+![Shipping Method Details](../../media/realtime_shipping_details.png)
+
+* Configure the countries in this shipping method should be allowed in
```
Comment (eshanrnh) [inline 13/umbraco-commerce/reference/shipping/realtime-rate-shipping.md:29]:
```suggestion
12. Select the countries this shipping method should be allowed in.
13. Click **Save**.
```

**Review by eshanrnh — CHANGES_REQUESTED** [review]
Hey @mattbrailsford 👋

Thanks for the PR. I have suggested few changes based on the style guide. Kindly let me know if it doesn't make sense to you.

**Review by eshanrnh — APPROVED** [review]
Merging it in.

**Pull request merged**